README.md

```diff
@@ -26,6 +26,7 @@ Twitter              | Fediverse                           | Name
 @alina_yurenko       | @alina_yurenko@mastodon.online      | Alina Yurenko
 @bazlur_rahman       | @bazlur_rahman@mastodon.online      | Bazlur Rahman
 @bclozel             | @bclozel@mamot.fr                   | Brian Clozel
+@mjduigou            | @bondolo@mastodon.social            | Mike Duigou
```

Comment:
The sorting is by Twitter handle. May I ask you to move your entry to the correct position? Thx!